pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/data/DemoItem.kt

```diff
@@ -61,10 +66,13 @@ sealed class DemoItem(
                 )
                 .setDrmConfiguration(
                     licenseUri?.let {
-                        DrmConfiguration.Builder(C.WIDEVINE_UUID)
-                            .setLicenseUri(licenseUri)
-                            .setMultiSession(true)
-                            .build()
+                        DrmConfiguration.Builder(C.WIDEVINE_UUID).apply {
+                            setLicenseUri(it)
+                            setMultiSession(multiSession)
+                            licenseRequestHeaders?.let { headers ->
+                                setLicenseRequestHeaders(headers)
+                            }
+                        }.build()
```

Comment:
Why not make `licenseRequestHeaders` non-`null` (by making it default to an empty map), and change this to:
```suggestion
                                                DrmConfiguration.Builder(C.WIDEVINE_UUID)
                            .setLicenseUri(it)
                            .setMultiSession(multiSession)
                            .setLicenseRequestHeaders(licenseRequestHeaders)
                            .build()
```

Reply:
The DRmConfiguration.Builder create a default one. It is strange that `DrmConfig.getLicenseRequestHeaders` is nullable.

I changed the constructor to have a non-null property